inductiva/resources/machine_cluster.py

```diff
@@ -0,0 +1,78 @@
+"""Class to manage the MPI cluster in Google Cloud."""
+from absl import logging
+
+from inductiva.resources import machines_base
+
+
+class MPICluster(machines_base.BaseMachineGroup):
+    """Class to launch and manage an MPI cluster in Google Cloud.
+
+    A MPI cluster is a collection of homogenous machines all working together on
+    a common task given the configurations that are launched in Google Cloud.
+    Note: The cluster will be available only after calling 'start' method.
+    The billing will start only after the machines are started."""
+
+    def __init__(self,
+                 machine_type: str,
+                 num_machines: int = 1,
+                 disk_size_gb: int = 70,
+                 register: bool = True) -> None:
+        """Create a MPICluster object.
+
+        Args:
+            machine_type: The type of GC machine to launch. Ex: "e2-standard-4".
+              Check https://cloud.google.com/compute/docs/machine-resource for
+              information about machine types.
+            num_machines: The number of virtual machines to launch.
+            disk_size_gb: The size of the disk in GB, recommended min. is 60 GB.
```

Comment:
Missing documentation for the `register` argument.

Reply:
Will attempt to making it clear, but on the description of the method, since this is not supposed to be changed by the users. Eventually, this will need to be changed. But  that will be done on an extra step.